tests/test_text.py

```diff
@@ -29,6 +29,14 @@ def test_parse_yaml():
     assert set(o) == {"a", "b"}
 
 
+@pytest.mark.parametrize("yaml_url, expected", [("https://raw.githubusercontent.com/GeoscienceAustralia/dea-config/"\
```

Comment:
I'm not keen on tests that depend on external resources and URLs, but, I know it's kind of painful to do otherwise. So, okay...

Reply:
We have an established pattern for handling this, actually. I wasn't going to say anything, but I agree here.

Reply:
Here's the example of faking a URL with a local file: https://github.com/digitalearthafrica/deafrica-scripts/blob/main/deafrica/tests/test_chirps.py#L105